modules/claims_api/spec/requests/v2/veterans/power_of_attorney_new_poa_request_spec.rb

```diff
@@ -0,0 +1,175 @@
+              Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                              'power_of_attorney', 'request_representative', 'invalid_claimant_schema.json').read
+            end
+
+            it 'returns a meaningful 422' do
+              VCR.use_cassette('claims_api/mpi/find_candidate/valid_icn_full') do
+                mock_ccg(scopes) do |auth_header|
+                  detail = "If claimant is present 'address' must be filled in  with required fields addressLine1, " \
+                           'city, stateCode, country and zipCode'
+
+                  post request_path, params: request_body, headers: auth_header
+
+                  response_body = JSON.parse(response.body)['errors'][0]
+
+                  expect(response).to have_http_status(:unprocessable_entity)
+                  expect(response_body['title']).to eq('Unprocessable Entity')
+                  expect(response_body['status']).to eq('422')
+                  expect(response_body['detail']).to eq(detail)
+                end
+              end
+            end
+          end
+        end
+
+        context 'when the request data passes schema validation' do
+          context 'when no representative is found with the provided poaCode and registrationNumber' do
+            let(:request_body) do
+              Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                              'power_of_attorney', 'request_representative', 'invalid_poa.json').read
+            end
+
+            it 'returns a meaningful 404' do
+              mock_ccg(scopes) do |auth_header|
+                detail = 'Could not find an Accredited Representative with registration number: 999999999999 ' \
+                         'and poa code: AG3'
+
+                post request_path, params: request_body, headers: auth_header
+
+                response_body = JSON.parse(response.body)['errors'][0]
+
+                expect(response).to have_http_status(:not_found)
+                expect(response_body['title']).to eq('Resource not found')
+                expect(response_body['status']).to eq('404')
+                expect(response_body['detail']).to eq(detail)
+              end
+            end
+          end
+        end
+      end
+
+      context 'successful request' do
+        let(:request_body) do
+          Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                          'power_of_attorney', 'request_representative', 'valid.json').read
+        end
+
+        it 'responds with created status and the original request body' do
+          VCR.use_cassette('claims_api/mpi/find_candidate/valid_icn_full') do
+            mock_ccg(scopes) do |auth_header|
+              # the final return from BGS does not matter at this point in time, the calls need only to succeed
+              allow_any_instance_of(ClaimsApi::PowerOfAttorneyRequestService::Orchestrator)
```

Comment:
We don't really need to make new VCR cassettes for the BGS calls in the controller since the Orchestrator specs cover all of those calls